src/js/Html5Player.js

```diff
@@ -9,6 +9,8 @@ class Html5Player extends Meister.PlayerPlugin {
     constructor(config, meister) {
         super(config, meister);
 
+        this.config.enableKeyBoardShortcuts = (typeof this.config.enableKeyBoardShortcuts === 'undefined' ? true : this.config.enableKeyBoardShortcuts);
```

Comment:
I think it's weird that the config is called 'enable' while the default state is already enabled.